service/src/main/java/bio/terra/workspace/app/controller/ControlledAzureResourceApiController.java

```diff
@@ -499,7 +499,8 @@ public ResponseEntity<ApiCloneControlledAzureStorageContainerResult> cloneAzureS
             body.getName(),
             body.getDescription(),
             body.getName(),
-            body.getCloningInstructions());
+            body.getCloningInstructions(),
+            body.getPrefixesToClone().toArray(new String[0]));
```

Comment:
TBH, I just thought of it as an array of strings instead of a list (and the matching implementation using a List was just the easiest way to write it). But when I tried to convert it to a List of strings throughout just now I did hit a serialization issue (even using TypeReference which is supposed to be the way around this). I spent about 45 minutes trying to figure it out and looking at other examples, but no luck. Does it really bother you if it's an array instead of a List?

```
Failed to deserialize value '["analyses/"]' from JSON to type java.util.List<java.lang.String>
bio.terra.stairway.exception.JsonConversionException: Failed to deserialize value '["analyses/"]' from JSON to type java.util.List<java.lang.String>
	at app//bio.terra.stairway.FlightMap.get(FlightMap.java:117)

Reply:
Nope, makes sense, thanks for the clarification.